src/mvba/proposal.rs

```diff
@@ -1,16 +1,9 @@
-use super::hash::Hash32;
+
 
 use serde::{Deserialize, Serialize};
 
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
 pub struct Proposal {
```

Comment:
So is this struct still in use? can we remove it as well?

Reply:
No. Actually I deleted it in the next PR. still didn't commit it.